package.json

```diff
@@ -267,8 +267,8 @@
     "@metamask/notification-controller": "^3.0.0",
     "@metamask/obs-store": "^8.1.0",
     "@metamask/permission-controller": "^5.0.0",
-    "@metamask/polling-controller": "^1.0.1",
     "@metamask/phishing-controller": "^7.0.1",
+    "@metamask/polling-controller": "^1.0.1",
```

Comment:
lint:fix

Reply:
I don't think it should go in this PR, since this is happening in develop? Wdyt